recipes/doctest/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "doctest" %}
+{% set version = "2.3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://raw.githubusercontent.com/onqtam/doctest/{{ version }}/doctest/doctest.h
+    sha256: dc617544b70d88b1ab939f1e5cf365a3848d75635e05a68459d8a354e4a89aba
+  - url: https://raw.githubusercontent.com/onqtam/doctest/{{ version }}/LICENSE.txt
+    sha256: 2ab6583af88018c4c6cadf8363bf3625505d117ac279ac174d935d992bac3592
+
+build:
+  number: 0
+  script: "mkdir -p $PREFIX/include && cp doctest.h $PREFIX/include/doctest.h"  # [unix]
+  script: "md '%LIBRARY_INC%' && COPY doctest.h %LIBRARY_INC%\\doctest.h"  # [win]
+  skip: true  # [win]
```

Comment:
What's the reason for skipping Windows?

Reply:
I couldn't make it work. Windows CI error message was not helpful. Any help is welcome.

Reply:
It could be that the quotes `'` are an issue. 

I'll merge and would suggest that you copy these two commands into separate lines in a `bld.bat`. That should give you better error messages.